department-of-veterans-affairs/caseflow
# Fixing flaky test - APPEALS-57724

Resolves [APPEALS-57724](https://jira.devops.va.gov/browse/APPEALS-57724)

# Description
1. Added a timecop.freeze and freeze it to certain date. 
2. change the way the withdrawal date is currently being saved it used to be closed_at.to_datetime instead changed it to closed_at.to_date
## Acceptance Criteria
- should not flake 
## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Login as a VHAADMIN
3. Go to in_progress tab, edit any request issue and withdraw.
4. withdraw a request issue by selecting a specific date and time. save it and come back it should still be save date you selected. 


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
